x11-misc/revelation/Manifest

```diff
@@ -0,0 +1 @@
+DIST revelation-0.5.3.tar.xz 464036 BLAKE2B 052866f805b05a34a900e8876e7fa3e2c6c19bfc1a685da69033980f10f68abd4344a2286e2b2e6a16b97a3b97aa81482aff2acfea349f2f4c3ea7c0da91ad96 SHA512 0523c42de55dfe110a155046f231134f5d0b6d1f88c9b6ca3abeea1089d6fca934af63e56202caba046a5043756f76e26957824517ad84b17a967326b42cd79b
```

Comment:
Upstream seems to have released 0.5.4 meanwhile, let's update this PR to have 0.5.4 instead of 0.5.3.